README.md

```
@@ -12,31 +12,48 @@
 ### 초기 화면
 | 로그인 | 홈 |
 | :---: | :---: |
-|<img width="310" alt="image" src="https://github.com/user-attachments/assets/523596b1-a1dc-4deb-87e7-479b20cd8fdf" /> | <img width="310" alt="image" src="https://github.com/user-attachments/assets/216969d5-d367-4768-a9c6-4820d2c1ba00" /> |
+|<img width="310" alt="image" src="https://github.com/user-attachments/assets/b068258d-1f2a-4a20-902c-4e1cc4997aee" />| <img width="310" alt="image" src="https://github.com/user-attachments/assets/bc23fba8-d7cf-4152-9ddc-90367dbff22c" /> |
 
-### 온보딩 (현재 구현 중)
-| <img width="310" alt="image" src="https://github.com/user-attachments/assets/d9a543bf-0852-4466-a2ac-3aa92ca4cbf5" /> | <img width="310" alt="image" src="https://github.com/user-attachments/assets/fbdab48c-57ee-45ca-bf7d-996ba9a5aadb" /> | <img width="310" alt="image" src="https://github.com/user-attachments/assets/b80f7be7-97d4-427c-b992-a6174e9cde2c" /> |
+### 온보딩
+| <img width="310" alt="image" src="https://github.com/user-attachments/assets/58690d7a-7fde-4427-910e-591ae8911b7f" /> | <img width="310" alt="image" src="https://github.com/user-attachments/assets/59dc1706-3465-4c09-b256-3c42a2b432e0" /> | <img width="310" alt="image" src="https://github.com/user-attachments/assets/d7c8ff1e-5815-4e8e-bf4f-88433ac218a5" /> |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)